django/db/models/sql/compiler.py

```diff
@@ -135,9 +135,6 @@ def collapse_group_by(self, expressions, having):
         # include the primary key of every table, but for MySQL it is enough to
         # have the main table's primary key.
         if self.connection.features.allows_group_by_pk:
-            # The logic here is: if the main model's primary key is in the
-            # query, then set new_expressions to that field. If that happens,
-            # then also add having expressions to group by.
```

Comment:
Maybe add `# Determine if the main model's primary key is in the query.` for this block?